suite-common/redux-utils/src/extraDependenciesType.ts

```diff
@@ -8,6 +8,9 @@ import { NotificationEventPayload } from '@suite-common/notifications';
 import { ActionType, SuiteCompatibleSelector, SuiteCompatibleThunk } from './types';
 
 type StorageLoadReducer = (state: any, action: { type: any; payload: any }) => void;
+type StorageLoadTransactionsReducer = (state: any, action: { type: any; payload: any }) => void;
```

Comment:
return type shouldn't be void but never, that also wrong for `StorageLoadReducer`